tests/test_connector.py

```diff
@@ -22,46 +15,17 @@ def test_connectivity_connector(mock_py42_client):
 
 
 class TestCode42Connector(object):
```

Comment:
A lot of the tests got significantly simplified, notably due to no longer needing `mock_result_adder`.
Another thing I tried to do was have every test assert that success status code was set (or the failure one, if that's what was needed). Because of that, we were able to get rid of tests that only checked the status code and nothing else.